src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java

```diff
@@ -2008,6 +2093,86 @@ public void onWakeUp()
         {
             homeBuilding.onWakeUp();
         }
+
+        BlockPos spawn;
+        if (getBedLocation() != BlockPos.ORIGIN)
+        {
+            spawn = Utils.scanForBlockNearPoint(
+              world,
+              getBedLocation(),
+              4,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 4 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)